### DISCUSSION TITLE
traffic signs import from opendrive

### DISCUSSION THREAD

**Discussion by efeld** [carla-simulator/carla#4652] [Ideas]
Do you have plans to expand the map import process from OpenDrive to support traffic signs other than Stop and Yield?

**Comment by hitabm** [comment]
I have the same problem too! Because I'm trying to drive fast and it causes fallings or collisions in road turnings. So I would need to add some speed signs in turns to slow down the ego vehicle temporarily in the way of avoiding collisions.
